### ISSUE TITLE
[REQ] True 0% Alpha Background?

### ISSUE THREAD

**Issue by wheatbread** [dfherr/ThreatClassic2#111]
### What is the feature you would like requested?
Hey, I love this addon!

I had a small aesthetic request: 

Currently, if I set the background alpha to 0, it actually doesn't go fully transparent and you can still see a transparent white background.

Pictures to explain:

This is my details setup, and what I'm trying to go for appearance-wise with ThreatClassic2:

![Screenshot_1](https://user-images.githubusercontent.com/548824/123275433-2c2b3700-d4b9-11eb-9993-3ac3ddf39590.png)

This is what ThreatClassic2 looks like with the background alpha set to 0%:

![Screenshot_2](https://user-images.githubusercontent.com/548824/123275557-4402bb00-d4b9-11eb-9bc4-7015ca1e6abd.png)

**Comment by dfherr** [comment]
I'm actually curious how you got the bars to look like this to begin with :).

The background can be made completly transparent already. That white stuff has to be an artifact from the bar color itself. How did you set it so that it only shows a white line on the bottom?

**Comment by wheatbread** [comment]
Oh, I use this media pack: https://www.wowinterface.com/downloads/info15613-FerousMediaPack.html

One of the bars is transparent except for the line at the bottom.  I'm using the same texture for the Details one and it shows no background, which is why I assumed that the background was from ThreatClassic2.

**Comment by dfherr** [comment]
I think the problem is the non-configurable backdrop.

I will have an update soon that makes this configurable. If you set the backdrop to alpha 0 too, it should be fully transparent

**Comment by wheatbread** [comment]
Ahh, I see. 

Thank you very much, that sounds awesome!

**Comment by dfherr** [comment]
should be doable in v2.6

**Comment by robmonte** [comment]
@dfherr Backdrop settings seem to be a bit bugged to me unless I'm misunderstanding the settings.

First, the backdrop color is drawing _on top of_ the bar text. If I set the backdrop color to 100% opacity then the text is completely covered by whatever color I choose.
Next, if I set the backdrop color to 0% opacity, the backdrop turns solid black instead of transparent.
Finally, when I change the backdrop texture, the setting always says "TC2 Default". On top of that, it only sometimes properly changes the backdrop texture on the meter. Most of the time it does nothing.

**Comment by dfherr** [comment]
@robmonte honestly, i'm not so sure myself how these settings interact. I just named them how blizzard named and exposed them. I also don't understand why the backdrop is drawn above the text, but what I'm setting is what blizzard calls the backdrop:

https://github.com/dfherr/ThreatClassic2/blob/master/core/core.lua#L104-L127

There definitely is a difference between background and backdrop and the backdrop is drawn above the background. Again, the text interaction also weirded me out, but I thought exposing these settings to players is still a net gain.

Yea, I overlooked a small bug in the backdrop texture setting. It gives back the value of the edge texture rather than the backdrop texure. Will be fixed in next release.
